src/mmrelay/cli.py

```
@@ -75,49 +73,99 @@ def _get_logger() -> logging.Logger:
-    if logger is None:
-        logger = _logger
     if _logger is None:
         raise RuntimeError("Logger must be initialized")
     return _logger
 
 
+def __getattr__(name: str) -> Any:
+    """Handle lazy loading of the module-level logger."""
+    if name == "logger":
+        return _get_logger()
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)